src/test/kotlin/org/web3j/EmbeddedGenesisTest.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.web3j.greeter.Greeter
+import org.web3j.protocol.Web3j
+import org.web3j.tx.TransactionManager
+import org.web3j.tx.gas.ContractGasProvider
+
+@EVMTest(type = NodeType.EMBEDDED, genesis = "file:src/test/resources/embedded/genesis.json")
```

Comment:
I don't see anything asserting that a genesis file was loaded. When loading a custom genesis file something should be different from the default option. Your test will need to assert on this difference.

Reply:
The way i knew that the genesis file is loaded is via providing a genesis file containing a typo vs a correct one...
I guess i can add a specific field in the genesis file, and ask for it in the test and see if i get the correct value.
What you think ?

Reply:
Yep, give that a go :)

Reply:
Hello,
Im trying to find a specific field that i can check for the test but i cant find any. 
In fact, after checking the `EmbeddedWeb3jService` in `web3j-evm`, the set of methods i can use is limited and the ones available are transactions oriented. However, i want, for example, to check the ChainID or the difficulty or... there doesnt seem to be a method offering that.
Any ideas ?

Reply:
Maybe your test genesis file can define a specific account with some amount of ETH in it, then your test can check if accounts exists with this amount in it?

Reply:
Great idea, thank you

Reply:
Done. What you think?

Reply:
Looking good